### DISCUSSION TITLE
How to make the output collapsible in jupyter-notebook using quarto?

### DISCUSSION THREAD

**Discussion by ANP-Oxy** [quarto-dev/quarto-cli#5193] [Q&A] [answered]
Hello everyone, I only started using Quarto today and I'm having some trouble with figuring out how to make the output of a jupyter code cell collapsible in Quarto. 

this is how I have setup the defaults
![image](https://user-images.githubusercontent.com/94713251/232093437-97511f99-3dde-4b15-9b58-679a185ead33.png)

I have this code where the output is pretty long
![image](https://user-images.githubusercontent.com/94713251/232092982-644293f2-eb29-4b51-8dbe-1d16a4960d7e.png)

when quarto renders it, it shows the whole output which doesn't look good. I want the reader to have the option collapse the output. just like there is a option to toggle the code is there a way to do same for the output?
![image](https://user-images.githubusercontent.com/94713251/232093098-c03e2790-b52a-49d8-98df-99315d317a0a.png)

**Comment by mcanouil** [comment, marked as answer]
See #5131, #4911 and the issue related to this #341.

**Comment by ANP-Oxy** [comment]
Thankyou.
